main.py

```diff
@@ -1,7 +1,39 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre
+from db.models import Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western",
+              "Action",
+              "Dramma"]
+    actors = ["George Klooney",
+              "Kianu Reaves",
+              "Scarlett Keegan",
+              "Will Smith",
+              "Jaden Smith",
+              "Scarlett Johansson"]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        Actor.objects.create(
+            first_name=actor.split()[0],
```

Comment:
you can loop over actors.split() with name, surname. So u don't need to split actors every time.
Or you can put actors in list of tuples E.g.  actors = [("George", "Klooney"),]
and loop with name, surname without split()

Reply:
idk why i don`t do this at first solution, ty <3